fiboa_cli/datasets/fieldscapes_austria_2021.py

```diff
@@ -0,0 +1,140 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/work/labwork_hkerner/fieldscapes/austria/boundaries_austria_2021.gpkg"
+
+# Unique identifier for the collection
+ID = "fieldscapes_austria_2021"
+# Title of the collection
+TITLE = "Field boundaries for Austria (Fieldscapes)"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Austria."""
```

Comment:
```suggestion
DESCRIPTION = "The dataset contains field boundaries for the Austria."
```

Reply:
Why does the template contain three apostrophes, if one is desired here? Is it that three lets you do multiline? But multiline isn't needed for this one?

Reply:
Yes, """ is multiline, " is single line.
I provided """ as default to make like simpler for implementors as I had hoped for a bit longer, usually multiline descriptions.